src/components/types.ts

```diff
@@ -139,7 +139,7 @@ export interface TableFormat extends DataFormat {
 export type TableData = TableFormat & Updates<TableFormat>
 
 type VegaLiteOptions = EmbedOptions<string, Renderers>
-export interface VegaLiteChartFormat extends DataFormat {
+export interface VegaLiteVizFormat extends DataFormat {
```

Comment:
Maybe this should just be `VegaLiteFormat` 
```suggestion
export interface VegaLiteFormat extends DataFormat {
```